app/main.py

```diff
@@ -1,2 +1,55 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    LOAD_FACTOR = 2 / 3
+
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.length = 0
+        self.threshold = int(self.capacity * self.LOAD_FACTOR)
+        self.hash_table = self.make_hash_table()
+
+    def make_hash_table(self) -> list:
+        return [[] for _ in range(self.capacity)]
+
+    def resize(self) -> None:
+        old_hash_table = self.hash_table
+        self.length = 0
+        self.capacity *= 2
+        self.threshold = int(self.capacity * self.LOAD_FACTOR)
+        self.hash_table = self.make_hash_table()
+        for element in old_hash_table:
+            if len(element) != 0:
```

Comment:
For lists, `len(list) != 0` is equivalent to `not list`. I'd suggest to change this check to the latter.